smarts/core/smarts.py

```diff
@@ -1385,13 +1385,21 @@ def _try_emit_envision_state(self, provider_state: ProviderState, obs, scores):
                 veh_type = (
                     v.vehicle_config_type if v.vehicle_config_type else v.vehicle_type
                 )
-                traffic[v.vehicle_id] = envision_types.TrafficActorState(
-                    actor_type=envision_types.TrafficActorType.SocialVehicle,
-                    vehicle_type=veh_type,
-                    position=tuple(v.pose.position),
-                    heading=float(v.pose.heading),
-                    speed=v.speed,
-                )
+
+                SUPPORTED_VEHICLES = {"car", "truck", "passenger", "bus", "trailer", "coach"}
```

Comment:
Since this is about what _envision_ supports, this should be defined in `envision_types` and exported from there.  (The `SMARTS` class should not be declaring what the Envision tool supports.)

However, there is already a `VehicleType` enum that can be imported from `envision_types`.  We should just use that here.  I'm pretty sure you can check for membership in a python enum type, although I don't know how off the top of my head.